src/Twig/Extensions.php

```diff
@@ -141,9 +143,29 @@ public function getFilters()
                 'orTemplateIfNotExists',
                 [$this, 'doesTwigFileExist']
             ),
+            new \Twig\TwigFilter(
+                'asXml',
```

Comment:
I'm not super happy about this name, what is a good name for this? It just formats XML, name turn it into XML.

Reply:
xmlFormatter maybe?